src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```
@@ -409,6 +409,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <ExecProjects Include="$(MSBuildThisFileDirectory)ExecWrapper.proj">
         <AdditionalProperties>
           ExecCommand=$(WasmCompilerPath) &quot;%(LlvmObjects.Identity)&quot; -o &quot;%(LlvmObjects.NativeObject)&quot; $(CompileWasmArgs)
```

```suggestion
          ExecCommand=$(WasmCompilerPath) &quot;%(LlvmObjects.Identity)&quot; -o &quot;%(LlvmObjects.NativeObject)&quot; $(CompileWasmArgs);
```
It needs a change in `ExecWrapper.proj` as well, to pass `$(ExecWorkingDirectory)` to the underlying task.

---

Sorry, I got a bit lazy there.